episodes/chap5_python_client.md

```diff
@@ -35,75 +73,352 @@ server_api = "/api"
+     \ V / (_| | | | | || (_| | (_| |  __/ (_) |
+      \_/ \__,_|_| |_|\__\__,_|\__, |\___|\___/
+                                __/ |
+                               |___/
+
+    --> Join us on Discord! https://discord.gg/rwRvwyK
+    --> Docs: https://docs.vantage6.ai
+    --> Blog: https://vantage6.ai
+    ------------------------------------------------------------
+    Cite us!
+    If you publish your findings obtained using vantage6,
+    please cite the proper sources as mentioned in:
+    https://vantage6.ai/vantage6/references
+    ------------------------------------------------------------
+
+    ...
+
+    --> Succesfully authenticated
+    ```
+
+:::::::::::::::::::::::::::::::::
+
+## Using the client
+
+The `Client` instance offers a set of attributes that correspond to the
+[vantage6 server resources (Episode 3)](./chap3_run_analysis_ui.md#vantage6-user-interface-basics)
+also described in the [official documentation (docs.vantage6.ai)](https://docs.vantage6.ai/en/main/introduction/introduction.html#vantage6-resources). The available attributes are:
+
+| Resource              | Description                                                                 |
+|-----------------------|-----------------------------------------------------------------------------|
+| `client.user`         | Manage users including your own user details                           |
+| `client.organization` | Manage organizations or the organization that you are part of|
+| `client.rule`         | View all available permission rules |
+| `client.role`         | Manage roles (are collections of rules) |
+| `client.collaboration`| Manage collaborations |
+| `client.task`         | Create new tasks and view their run data |
+| `client.result`       | Obtain results from the tasks |
+| `client.util`         | Provides utility functions for the vantage6 Python client. For example to reset your password.             |
+| `client.node`         | Manage nodes |
+| `client.store`        | Manage an algorithm store |
+| `client.algorithm`    | Manage algorithms that can be used for the computations |
+
+Almost all of these attributes provide a [get](#get), [list](#list), [create](#create),
+[update](#update) and [delete](#delete) operation. All of the functions that interact
+with the server return a dictionary with the requested information.
+
+
+::: callout
+
+### Permissions
+
+Note that the authenticated user may not be allowed to perform all operations or view all
+resources. The server will only allow the user to perform operations on the resources
+that the user has permission to perform.
 
-Each of these attributes, in turn, provides an abstraction of the CRUD (Create, Read, Update, and Delete) operations that can be performed to its corresponding concept\*:
+For example, a user may not be allowed to create a new organization, but may be allowed
+to list the organizations within all collaboration its organization participates in.
```

Comment:
The example refers to the first line of the first paragraph, so I would change the order of the sentences and put the example in the middle of the paragraph above